src/main/asciidoc/tasks/file-ingest/main.adoc

```diff
@@ -65,7 +66,7 @@ dataflow:>
 . Launch the task
 +
 ```
-dataflow:>task launch fileIngestTask --arguments "filePath=classpath:data.csv"
+dataflow:>task launch fileIngestTask --arguments "filePath=classpath:data.csv --spring.cloud.task.closecontext_enable=false"
```

Comment:
Isn't this `false` by default based on [this](https://github.com/spring-cloud/spring-cloud-task/commit/94e074d8412b9cf3163c125292de2d3d5858f2c7#diff-56a27258e7cac74fee72450b7840fb8e)?

Maybe we aren't using the updated version here?

Reply:
yeah, I see this uses `1.2.2.RELEASE`.